avanza/avanza.py

```diff
@@ -98,14 +98,24 @@ def __call(self, method: HttpMethod, path: str, options=None):
         if method_call is None:
             raise ValueError(f'Unknown method type {method}')
 
-        response = method_call(
-            f'{BASE_URL}{path}',
-            json=options,
-            headers={
-                'X-AuthenticationSession': self._authentication_session,
-                'X-SecurityToken': self._security_token
-            }
-        )
+        if method == HttpMethod.GET:
+            response = method_call(
+                f'{BASE_URL}{path}',
+                params=options,
+                headers={
+                    'X-AuthenticationSession': self._authentication_session,
+                    'X-SecurityToken': self._security_token
+                }
+            )
+        else:
+            response = method_call(
+                f'{BASE_URL}{path}',
+                json=options,
+                headers={
+                    'X-AuthenticationSession': self._authentication_session,
+                    'X-SecurityToken': self._security_token
+                }
+            )
```

Comment:
This can be simplified as:
```python
data = {}
if method == HttpMethod.GET:
    data['params'] = options
else:
    data['json'] = options

response = method_call(
    f'{BASE_URL}{path}',
    headers={
        'X-AuthenticationSession': self._authentication_session,
        'X-SecurityToken': self._security_token
    },
    **data
)
```

Reply:
Nice suggestion. I'll add that.